sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -66,7 +66,7 @@ public class ServiceBusReceiver : IAsyncDisposable
         /// Gets the ID to identify this client. This can be used to correlate logs and exceptions.
         /// </summary>
         /// <remarks>Every new client has a unique ID.</remarks>
```

Comment:
Here, again, we should be cautious about making this guarantee, since the value is not under our control.